pyproject.toml

```diff
@@ -39,6 +39,7 @@ dependencies = [
     "torchvision",
     "llvmlite>=0.40.0",
     "umap-learn",
+    "opencv-python",
```

Comment:
I have created a new set of dependencies for benchmarking in #888. These four dependencies will then move out of the main coreax dependencies.